documentation/config_file_puppetserver.md

```diff
@@ -90,6 +99,12 @@ http-client: {
 profiler: {
     enabled: true
 }
-~~~
+
+# Settings related to static catalogs.
+versioned-code: {
+    code-id-command: /opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh
+    code-content-command: /opt/puppetlabs/server/apps/puppetserver/code-content-command_script.sh
+}
+```
```

Comment:
Do we ever provide puppet code for managing these using the [puppetlabs-hocon](https://forge.puppetlabs.com/puppetlabs/hocon) module?

```puppet
hocon_setting { 'code-id-command':
  ensure  => present,
  path    => '/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf',
  setting => 'versioned-code.code-id-command',
  value   => '/path/to/code-id.sh',
}

hocon_setting { 'code-content-command':
  ensure  => present,
  path    => '/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf',
  setting => 'versioned-code.code-content-command',
  value   => '/path/to/code-content.sh',
  ]
}
```